src/JSTest/RuntimeManager.cs

```diff
@@ -44,6 +43,11 @@ public TestRuntimeManager(JSTestSettings settings, TestRunEvents testRunEvents)
             }
         }
 
+        public int getProcessId()

```

Comment:
>getProcessId [](start = 19, length = 12)

nitpick: fix the casing.. and rename the function to indicate which process..

Reply:
fixed